.gitignore

```diff
@@ -34,6 +34,8 @@ build/_test
 build/_maven_output
 build/_maven_overlay
 build/_kamelets
+build/_maven_overlay/
```

Comment:
Do we really need this twice?  Does the last `/` change anything?

Reply:
I probably overlooked the directory was already there, I will push another change to fix this, thanks for reviewing.